.changeset/red-candles-invite.md

```diff
@@ -0,0 +1,21 @@
+---
+'@astrojs/sitemap': minor
+---
+
+Add a `customSitemaps` option to include extra sitemaps in the `sitemap-index.xml` file generated by Astro. This is useful for multi-framework setups on the same domain, where for example Astro generates `example.com` and some other framework handles `example.com/blog` and you want to include `example.com/blog/sitemap.xml` to `sitemap-index.xml`.
+
+Example:
+
+```js
+import { defineConfig } from 'astro/config';
+import sitemap from '@astrojs/sitemap';
+
+export default defineConfig({
+  site: 'https://example.com',
+  integrations: [
+    sitemap({
+      customSitemaps: ['https://example.com/blog/sitemap.xml', 'https://example.com/helpcenter/sitemap.xml'],
+    }),
+  ],
+});
+```
```

Comment:
``````suggestion
```

Learn more in the [`@astrojs/sitemap` configuration documentation](https://docs.astro.build/en/guides/integrations-guide/sitemap/#configuration).
``````
Just adding some kind of link back to documentation! This could go directly to the entry itself (`/#customsitemaps`) or just pointing to config options is fine, I think!

Reply:
Hi, just a note that if you would like to accept this change, you can use the "commit suggestion" button to update the content of the file! Right now, this sits as an unresolved issue until you either accept my suggestion, or ignore it and update the code yourself. :smile: 

Like in the comment above, you are welcome to simply make changes yourself and I can always check it over for language again!